src/modules/APIs.js

```diff
@@ -0,0 +1,17 @@
+import { foods } from './food';
+
+const commentkey = 'HJ3nJrQxKFWMKnh6RtwV';
+const commentUrl = `https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/${commentkey}/comments`;
+// const  = `${commentApi}/apps/${commentkey}/comments/`;
```

Comment:
- [Optional] Please remove all commented code from your all documents. This will make your document neater.

Reply:
Okay, I will do that now.